api/src/api.ts

```diff
@@ -168,23 +169,36 @@ export const plugin = {
     server.route({
       method: "GET",
       path: "papers/{paperSelector}/entities",
-      handler: async (request) => {
+      handler: async (request, h) => {
         const paperSelector = parsePaperSelector(request.params.paperSelector);
         // Runtime type-checked during validation.
         const entityTypes = request.query.type as EntityType[];
-        let res;
+        // pretty sure Joi is actually making this a boolean, despite Hapi saying it's a string
+        const slim = !!request.query.slim;
+        let res: Entity[] = [];
         try {
-          res = await dbConnection.getEntitiesForPaper(paperSelector, entityTypes);
+          res = await dbConnection.getEntitiesForPaper(paperSelector, entityTypes, slim);
```

Comment:
When you reconcile against my branch you'll want to take a peek at this -- I have a new boolean arg to this method as well.